flink-clients/src/test/java/org/apache/flink/client/cli/CliFrontendSavepointTest.java

```diff
@@ -282,12 +279,14 @@ public void testDisposeSavepointFailure() throws Exception {
 
 	// ------------------------------------------------------------------------
 
-	private static final class DisposeSavepointClusterClient extends StandaloneClusterClient {
+	private static final class DisposeSavepointClusterClient extends RestClusterClient<StandaloneClusterId> {
```

Comment:
Does this remove test coverage for the old client?

Reply:
see also reply on #7640